src/hotspot/share/opto/superword.cpp

```diff
@@ -2790,6 +2919,17 @@ bool SuperWord::output() {
         return false; // bailout
       }
 
+#ifdef ASSERT
+      // Mark Load/Store Vector for alignment verification
+      if (AlignVector && VerifyAlignVector && cl->is_main_loop()) {
+        if (vn->Opcode() == Op_LoadVector) {
+          vn->as_LoadVector()->set_must_verify_alignment();
+        } else if (vn->Opcode() == Op_StoreVector) {
```

Comment:
Do you mark all Load/StoreVector nodes for verification? Then why do you even need to set such field?
I you verify only few you need comment explaining which one you are verifying.

Reply:
I will add a comment. We only mark the nodes created by SuperWord, because AlignVector does only apply to SuperWord, and not for example to VectorAPI nodes.